### PR TITLE
Adding boot order manipulation and few other changes

### PR DESCRIPTION
**Pull request by smalleni**

### CODE REVIEW

File: tasks/setup_infrared.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@
   git:
       repo: "https://github.com/redhat-openstack/infrared"
       dest: "{{ infrared_dir }}"
+      force: yes
```
Comment (venkataanil) [inline tasks/setup_infrared.yml:6]: We don't want to forcefully clone if it exists.

File: group_vars/all.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+controller_count: 3
+# No need to set compute_count. This will be set to all remaining nodes, which is calclulated in overcloud.yml
+#compute_count: 1
+alias:
+#lab specific vars
+  lab_url: "http://quads.alias.bos.scalelab.redhat.com"
+scale:
+# lab specific vars
+  lab_url: "http://quads.rdu2.scalelab.redhat.com"
+machine_types:
```
Comment (venkataanil) [inline group_vars/all.yml:22]: We can get vendor name by reading sys entries
[root@f16-h10-000-1029p ~]# cat /sys/class/dmi/id/board_vendor
Supermicro
[root@e23-h33-740xd ~]# cat /sys/class/dmi/id/board_vendor
Dell Inc.
[vkommadi@localhost ~]$ cat /sys/class/dmi/id/board_vendor
LENOVO

But this change is more efficient which allows to find the vendor without login into the machine.

**Review by venkataanil — COMMENTED** [review]
Thanks for the patch. Want to merge if you remove the "force" option for cloning infrared.

File: group_vars/all.yml (hunk 2 above)
Comment (venkataanil) [reply group_vars/all.yml:22]: I mean your change (i.e getting vendor by parsing url) is more efficient :)

**Comment by smalleni** [comment]
Closing as no longer relevant. Future commits will address this.

**Pull request closed without merging**